src/main/java/org/jabref/gui/Base.css

```
@@ -265,6 +265,8 @@
     -jr-ai-message-user-border: -jr-theme;
     -jr-ai-message-ai: -jr-accent;
     -jr-ai-message-ai-border: -jr-theme;
+    -jr-ai-message-error: -jr-light-red;
```

I assume, it should have the same color as other error messages. Therefore, re-use the CSS class:

```suggestion
    -jr-ai-message-error: -jr-error;
```